doc/manipulating_visualizing/data_preparation.rst

```diff
@@ -436,3 +435,32 @@ possible option.
 
    * :ref:`sphx_glr_auto_examples_connectivity_plot_probabilistic_atlas_extraction.py`
 
+Extraction of signals from seeds:\  :class:`NiftiSpheresMasker`.
+==================================================================
+
+The purpose of :class:`NiftiSpheresMasker` is to compute signals from
+seeds containing voxels in spheres. It makes it easy to get these signals once
+you have a list of coordinates.
+
+Seeds definition
+------------------
+
+A single seed is a sphere defined by the radius (in millimeters) and the
+coordinates (typically MNI or TAL) of its center.
+
+A set of seeds is thus defined by a list of 3D coordinates and a radius.
+Seeds can overlap spatially and are represented in a binary present-nonpresent
+coding (no weighting).
+
+:class:`NiftiSpheresMasker` Usage
+---------------------------------
+
+"seeds" argument takes a list of 3D coordinates of the spheres centers,
+they should be in the same space as the images (typically MNI or TAL).
```

Comment:
I agree. I tried to follow the previous sections but two paragraphs might be enough.